src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/resthandler/RestChangePolicyActionIT.kt

```diff
@@ -392,32 +380,26 @@ class RestChangePolicyActionIT : IndexStateManagementRestTestCase() {
 
     fun `test change policy API should only apply to indices in the state filter`() {
         val thirdState = randomState(actions = emptyList(), transitions = emptyList())
-        val secondState = randomState(actions = emptyList(), transitions = listOf(Transition(thirdState.name, null)))
+        val secondState = randomState(actions = listOf(randomReplicaCountActionConfig()), transitions = listOf(Transition(thirdState.name, null)))
```

Comment:
Any reason these states were defined in reverse order (ex. third...first, not first...third)?

Reply:
thirdState is used in secondState which is used in firstState.